### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by jimenezfede**

### CODE REVIEW

**Review by wilsuriel03 — APPROVED** [review]

File: server/routes/badges/badge.controller.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const postBadge = (req: Request, res: Response) => {
     }
   })
     .then((data) => {
-      console.log('badge created', data);
+      // console.log('badge created', data);
```
Comment (iamdestinos) [inline server/routes/badges/badge.controller.ts:18]: likely remove

File: server/routes/images/images.controller.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ const uploadImage = async (req: Request, res: Response) => {
   const command = new PutObjectCommand(params.send);
   s3.send(command)
     .then((data) => {
-      console.log('s3 command successful', data);
+      // console.log('s3 command successful', data);
```
Comment (iamdestinos) [inline server/routes/images/images.controller.ts:41]: should likely remove

File: server/routes/shows/shows.controller.ts [hunk 3]
```diff
@@ -191,6 +191,7 @@ const saveShow = async (req: Request, res: Response) => {
           isSaved,
         },
       });
+      console.log(save);
```
Comment (iamdestinos) [inline server/routes/shows/shows.controller.ts:194]: should likely edit to be shortened or removed entirely

**Review by iamdestinos — APPROVED** [review]
Comment Context shouldn't be here but is for some reason? otherwise approved

**Pull request merged**